src/components/Nav/Nav.js

```diff
@@ -1,10 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import NavModal from 'components/NavModal/NavModal';
+import './Nav.scss';
+import { BsHeart } from 'react-icons/bs';
+import { BsPerson } from 'react-icons/bs';
+import { BsCart2 } from 'react-icons/bs';
+import { BsSearch } from 'react-icons/bs';
```

Comment:
import 순서 
```suggestion
// external
import React, { useState } from 'react';
import { Link } from 'react-router-dom';
// npm dependency
import { BsHeart } from 'react-icons/bs';
import { BsPerson } from 'react-icons/bs';
import { BsCart2 } from 'react-icons/bs';
import { BsSearch } from 'react-icons/bs';
// internal
import NavModal from 'components/NavModal/NavModal';
// style
import './Nav.scss';
```
순으로 바꾸면 좋을 것 같습니다.

팀 협업을 위한 import order rule이 있다면 거기에 맞춰 정렬해주시면 좋을 것 같아요!

만약 import order 까지 신경쓰기 귀찮으시다면 아래 링크 참고하셔서 eslint에 import order rule을 추가하여 자동화 해도 좋을 것 같군요

- [eslint-plugin-import npm](https://www.npmjs.com/package/eslint-plugin-import)
- [eslint-plugin-import readme](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md)
- [관련 블로그](https://seohyun0120.tistory.com/m/entry/ESLint-importsexports-%EA%B5%AC%EB%AC%B8%EC%9D%98-%EC%88%9C%EC%84%9C-%EC%9E%90%EB%8F%99-%EC%A0%95%EB%A0%AC%ED%95%98%EA%B8%B0)